README.md

```diff
@@ -4,7 +4,7 @@ This repository holds the official specifications and reference implementations
 for WebAssembly.
 
 At this time, the contents of this repository are under development and known
-to be incomplet and inkorrect.
```

Comment:
These typos had been intentional, referencing a [somewhat](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2015/n4399.html) [common](http://gambit-list.iro.umontreal.narkive.com/cJycey10/incomplet-and-or-inkorrect-implementation-of-srfi-9) tradition (which is also a little humorous) in specs.